monitoring/monit.py

```diff
@@ -119,7 +120,7 @@ def run_command(command):
         if module.check_mode:
             module.exit_json(changed=True)
         status = run_command('unmonitor')
-        if status in ['not monitored']:
+        if status in ['not monitored'] or 'unmonitor pending' in status:
```

Comment:
I think "unmonitor pending" can be combined with other status not only "not monitored", so "startWith" could work but I don't know if it works always.
I choosed this kind of approach because is the same used in the other commands in monit module: a list of statuses or the action is pending.